# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/Caches.h
# Commit: 36780658b33a
# Full Hash: 36780658b33a050e82385161eab2e5e2e5e90064
# Author: Tooru Fujisawa <arai_a@mac.com>
# Date: 2024-12-05 09:58:04
# Regressor Bug: 1933681
# File Overlap Count: 1
# Description:
#   Bug 1933681 - Part 2: Rewrite concurrent delazification's input/output based on InitialStencilAndDelazifications. r=nbp
#   
#   The concurrent delazification internal is still using CompilationStencilMerger.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D230388
# ==============================================================================

diff -r 31d3f2901db9 -r 36780658b33a js/src/vm/Caches.h
--- a/js/src/vm/Caches.h	Wed Dec 04 23:44:33 2024 +0000
+++ b/js/src/vm/Caches.h	Wed Dec 04 23:44:33 2024 +0000
@@ -20,7 +20,6 @@
 #include "js/TypeDecls.h"
 #include "vm/JSScript.h"
 #include "vm/Shape.h"
-#include "vm/StencilCache.h"  // js::DelazificationCache
 #include "vm/StringType.h"
 
 namespace js {
@@ -649,15 +648,9 @@
 #endif
   }
 
-  void purgeStencils() {
-    DelazificationCache& cache = DelazificationCache::getSingleton();
-    cache.clearAndDisable();
-  }
-
   void purge() {
     purgeForCompaction();
     uncompressedSourceCache.purge();
-    purgeStencils();
   }
 };
 